src/platform/options/default_configuration.cpp

```diff
@@ -57,7 +57,7 @@ char const* const mo::composite_delay_opt         = "composite-delay";
 char const* const mo::enable_key_repeat_opt       = "enable-key-repeat";
 char const* const mo::x11_display_opt             = "x11-display-experimental";
 char const* const mo::wayland_extensions_opt      = "wayland_extensions";
-char const* const mo::wayland_extensions_value    = "wl_shell:xdg_wm_base:zxdg_shell_v6";
+char const* const mo::wayland_extensions_value    = "wl_shell:xdg_wm_base:zxdg_shell_v6:layer_shell_v1";
```

Comment:
I know I have some guilt in this (I didn't make the way to do it obvious), but we should probably not enable layer_shell_v1 by default.

We need a real implementation (and tests to prove it) before enabling it by default. As it clearly affects window management, we probably need to surface it through miral and let the window manager participate in the decision to enable this.